overrides/config/advRocketry/planetDefs.xml

```diff
@@ -112,4 +112,5 @@
 			<rotationalPeriod>21100</rotationalPeriod>
         </planet>
 		</star>
-</galaxy>
\ No newline at end of file
+    <star name="Eye Of Eternity" numGasGiants="0" numPlanets="0" size="2.0" temp="800" x="300" y="120" blackHole="true" > </star>
```

Comment:
This should be a tab indentation, to fit with the rest of the document.

Reply:
My bad, my editor uses spaces instead of tabs which I will change now. Thanks for the heads up!